src/core/main.c

```diff
@@ -294,6 +295,18 @@ static int parse_crash_chvt(const char *value) {
         return 0;
 }
 
+static int parse_confirm_spawn(const char *value, char **console) {
+        int r;
+
+        r = parse_boolean(value);
+        if (r == 0)
+                value = NULL;
+        else if (r > 0)
+                  value = "/dev/console";
+
+        return free_and_strdup(console, value);
```

Comment:
hmm, i think it would be nice to prepend "/dev" here, if it's not specified, so that systemd.confirm_spawn=ttyS0 is sufficient for this to work...